activerecord/lib/active_record/normalization.rb

```diff
@@ -5,11 +5,13 @@ module Normalization
     extend ActiveSupport::Concern
 
     included do
-      class_attribute :normalized_attributes, default: Set.new
-
-      before_validation :normalize_changed_in_place_attributes
+      include ActiveModel::Normalization
     end
 
+    ##
+    # :method: normalize_attribute
+    # :call-seq: normalize_attribute(name)
+    #
     # Normalizes a specified attribute using its declared normalizations.
```

Comment:
I'm concerned that the docs for all these `:call-seq:` methods will become out of sync with `ActiveModel::Normalization`.  Perhaps it would be better to merely point to the underlying method, like:

```ruby
# See ActiveModel::Normalization::ClassMethods#normalize_attribute.
```